ui/components/app/signature-request/signature-request-data/signature-request-data.stories.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import SignatureRequestData from './signature-request-data';
+
+export default {
+  title: 'Components/App/SignatureRequestData',
```

Comment:
Title should follow folder structure after `ui/` so if the story lives in  `ui/components/app/signature-request/signature-request-data/signature-request-data.stories.js` then the title should be
```suggestion
  title: 'Components/App/SignatureRequest/SignatureRequestData',
```

Reply:
If you ever need storybook boiler plate there is a quick example in the storybook docs. I use it quite a lot 😁  https://metamask.github.io/metamask-storybook/?path=/story/getting-started-documentation-guidelines--page#creating-a-story

<img width="1440" alt="Screenshot 2022-12-08 at 11 22 09 AM" src="https://user-images.githubusercontent.com/8112138/206548347-035662fd-b6f9-4ff4-aea9-6f069a42d622.png">

Reply:
@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/16759/commits/50f1359cb4bb5220486518775005f7d4e9993c8d).